content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
+1. Clone the documentation repository (https://github.com/sodafoundation/documentation.git) 
+2. Navigate to the folder structure content/support-matrix/delfin
+3. Create a new folder and follow the naming format `vendor`. For example `dell-emc`. 
```

Comment:
This line is not very clear. esp. "naming format `vendor`"

Reply:
changed. done.